docs/docsite/rst/scenario_guides/guide_meraki.rst

```diff
@@ -0,0 +1,128 @@
+.. _meraki_guide:
+
+Cisco Meraki Guide
+==================
+
+
+.. _meraki_guide_intro:
+
+What is Cisco Meraki?
+---------------------
+
+Cisco Meraki is an easy to use, cloud based, network infrastructure platform for enterprise environments. While most network hardware uses command line interfaces (CLIs) for configuration, Meraki uses an easy to use Dashboard hosted in the Meraki cloud. No on-premises management hardware or software is required - only the network infrastructure to run your business.
```

Comment:
This line needs a couple of hyphens:
cloud-based
command-line
easy-to-use

Reply:
@acozine JFYI you can edit that right in GitHub UI rather than waiting for the contributor. It's just faster sometimes :)

Reply:
Fixed.